pom.xml

```diff
@@ -22,11 +22,11 @@
     <parent>
         <groupId>org.apache.sling</groupId>
         <artifactId>sling</artifactId>
-        <version>32</version>
+        <version>33-SNAPSHOT</version>
```

Comment:
33 has been released, please don't rely on snapshots for parent versions (https://issues.apache.org/jira/browse/INFRA-15815)

Reply:
Thanks. Did not know this. I used the version from the jcr oak server bundle. I will update my PRs on both projects to use the release version.